src/common.ts

```diff
@@ -186,21 +186,21 @@ export function convertNativePathToPosix(pathString: string): string {
 }
 
 /**
- * This function checks to see if a subfolder is part of folder.
+ * This function checks to see if a subfolder is part of a parent folder.
  *
- * Assumes subfolder and folder are absolute paths and have consistent casing.
+ * Assumes subfolder and parent folder are absolute paths and have consistent casing.
  *
- * @param subfolder subfolder to check if it is part of the folder parameter
- * @param folder folder to check aganist
+ * @param subfolder subfolder to check if it is part of the parent folder parameter
+ * @param parentFolder folder to check aganist
  */
-export function isSubfolderOf(subfolder: string, folder: string): boolean {
+export function isSubfolderOf(subfolder: string, parentFolder: string): boolean {
```

Comment:
the previous version of this made no sense - it was checking that `folder` was a subfolder of `subfolder`.  I renamed the parameters to be more clear and reversed the check.

There was only one usage of this function in the O# workspace configuration provider.